### PR TITLE
feat(icon): CatalogIcon from Layer5 brand

### LINKED ISSUE #632: [Icon]: Add CatalogIcon from Layer5 brand page
#### Current Behavior
- There is no existing implementation for CatalogIcon from [brand page](https://layer5.io/company/brand) in sistent

#### Expected Behavior
- Add CatalogIcon into sistent, exported ready to use with color parameters.

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources

- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
  - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and [designs for Sistent site](https://www.figma.com/file/ekm0Hh6M25bSKpP5BcIJom/Sistent-Design-System?type=design&node-id=596%3A5227&mode=design&t=8MExPykHN1vQFCTZ-1) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by dottharun**
**Notes for Reviewers**

This PR fixes #632 

- [x] Adds to CatalogIcon from the brandpage.
- [x] dark mode support.
- [x] updated new Icon from Brand.

#### Screenshots tested in Meshery-UI:
image
image

**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**

- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by leecalcote — COMMENTED** [review]
Light mode and dark mode versions?

**Comment by leecalcote** [comment]
[catalog.zip](https://github.com/user-attachments/files/15594663/catalog.zip)

File: src/icons/CatalogIcon/CatalogIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  width = DEFAULT_WIDTH,
+  height = DEFAULT_HEIGHT,
+  primaryFill,
+  secondaryFill,
+  style = {},
+  ...props
+}) => {
+  const theme = useTheme();
+
+  const themePrimaryFill = primaryFill ?? (theme.palette.mode === 'dark' ? LIGHT_GRAY : KEPPEL);
```
Comment (leecalcote) [inline src/icons/CatalogIcon/CatalogIcon.tsx:21]: @nebula-aac is this representative of our approach to light and dark mode support?

**Review by leecalcote — APPROVED** [review]

**Comment by leecalcote** [comment]
LGTM

**Review by Yashsharma1911 — APPROVED** [review]

File: src/icons/CatalogIcon/CatalogIcon.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  secondaryFill,
+  tertiaryFill = WHITE,
+  style = {},
+  ...props
+}) => {
+  const theme = useTheme();
+  const themeMode = theme?.palette?.mode ?? 'light';
+
+  const themePrimaryFill = primaryFill ?? (themeMode === 'dark' ? KEPPEL : DARK_SLATE_GRAY);
+  const themeSecondaryFill = secondaryFill ?? (themeMode === 'dark' ? CARIBBEAN_GREEN : KEPPEL);
```
Comment (Yashsharma1911) [inline src/icons/CatalogIcon/CatalogIcon.tsx:25]: @sudhanshutech @aabidsofi19 is this the correct way of using theme in Sistent? @dottharun you are able to test this right?

Comment (dottharun) [reply src/icons/CatalogIcon/CatalogIcon.tsx:25]: Works in Meshery-UI when wrapped with <UsesSistent />.

Comment (Yashsharma1911) [reply src/icons/CatalogIcon/CatalogIcon.tsx:25]: Ohh thankyou for confirming

**Pull request merged**